Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -30,13 +34,24 @@
   desc: A slightly unstable medication used for the most extreme any serious case of radiation poisoning. Lowers radiation level at over twice the rate Hyronalin does and will heal toxin damage at the same time. Deals very minor brute damage to the patient over time, but the patient's body will typically out-regenerate it easily.
   physicalDesc: cloudy
   color: "#bd5902"
+  Metabolism:
+    - !type:HealthChangeMetabolism 
+      healthChange: -1
+      damageClass: Toxin #I think you need multiple of these components for different types of damage so I'll maybe change it to work better in the future 
```

Comment:
the way it works right now seems fine to me tbh, not many things will change multiple damageclasses at once and currently the separation makes it more obvious what's happening